src/workloads/replication/startup/README.md

```diff
@@ -32,9 +34,33 @@ The test is structured with several Genny workloads that are executed in order b
+  | 2. Stop checkpointing and add update CRUD operations.<br>(which controls the number of oplogs that will be replayed in startup recovery). | [Step_1_1_5GB_crud_50k_ops](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/1_1_5GB_crud.yml) | [Step_2_1_50GB_crud_10M_ops](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/2_1_50GB_crud.yml) |
+  | 3. Stop checkpointing and add DDL and CRUD operations.<br> It establishes a new database, generates new collections, builds secondary indexes, and proceeds to insert data. | [Step_1_2_5GB_ddl_55_storage_tables](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/1_2_5GB_ddl.yml) <br>- Creates **1 new database**. <br>- Creates **5 collections** with extra **10 indexes**. <br>- Inserts documents for an approximate total of **~1MB per collection**, **~5MB total**. | [Step_2_2_50GB_ddl_10k_storage_tables](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/2_2_50GB_ddl.yml) <br>- Creates **1 new database**. <br>- Creates **1000 collections** with extra **10 indexes**.<br>- Inserts documents for an approximate total of **100KB per collection**, **~1GB total**. |
+  | 4. Stop any IndexBuild from getting completed and start an IndexBuild that will continue after startup recovery.<br>(It drops all the 10 secondary indexes built in the first step and starts building them again which will get paused). | [Step_1_3_5GB_index](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/1_3_5GB_index.yml) | [Step_2_3_50GB_index](https://github.com/mongodb/genny/blob/1189cda0aa80e6104c7fe87016f1a3de98b18dba/src/workloads/replication/startup/3_0_Reads.yml) |
 
 To ensure DSI restarts mongod after the last workload, a dummy read workload has been added as the final workload.
+
+## Reading Results
+
+The time taken for startup recovery and shutdown is tracked in these two timing metrics: 
+| Metric | Description |
+| ---------- | ------------ |  
+| ShutdownTime | The time in milliseconds it takes to shutdown all launched mongod programs.	|
+| RecoveryTimeAfterRestart | The "recovery" time in milliseconds that it takes to start the server after this particular test phase / session ends. This metric is equal to the start time of the subsequent phase. |
+
+**Example**
+
+  | Test | Measurement | Patch Result |
+  | -------- | -------- | -------- |
+  | Step_1_1_5GB_crud_50k_ops Timing Metrics | ShutdownTime0 | 2605.681189 |
+  | Step_1_1_5GB_crud_50k_ops Timing Metrics | RecoveryTimeAfterRestart0 | 98552.713478 |
+  | Step_2_1_50GB_crud_10M_ops Timing Metrics | ShutdownTime0 | 11425.647337 |
+  | Step_2_1_50GB_crud_10M_ops Timing Metrics | RecoveryTimeAfterRestart0 | 960575.04176 |
```

Comment:
Is there still a Step 3? What does that one mean?

Reply:
step 3 is the dummy read.

It is doing 10 million crud operations on the 50GB collection.

Reply:
How should I interpret the metrics for Step 3?

Reply:
We should ignore it at all, I will add a line to indicate that.